composer.json

```diff
@@ -7,40 +7,40 @@
     "require": {
         "php": "^7.3 || ^8.0",
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/installer": "4.x-dev",
-        "silverstripe/recipe-authoring-tools": "1.x-dev",
-        "silverstripe/recipe-blog": "1.x-dev",
-        "silverstripe/recipe-collaboration": "1.x-dev",
-        "silverstripe/recipe-form-building": "1.x-dev",
-        "silverstripe/recipe-reporting-tools": "1.x-dev",
-        "silverstripe/recipe-services": "1.x-dev",
-        "silverstripe/recipe-content-blocks": "2.x-dev",
-        "silverstripe/recipe-solr-search": "2.x-dev",
+        "silverstripe/installer": "4.10.x-dev",
+        "silverstripe/recipe-authoring-tools": "1.10.x-dev",
+        "silverstripe/recipe-blog": "1.10.x-dev",
+        "silverstripe/recipe-collaboration": "1.10.x-dev",
+        "silverstripe/recipe-form-building": "1.10.x-dev",
+        "silverstripe/recipe-reporting-tools": "1.10.x-dev",
+        "silverstripe/recipe-services": "1.10.x-dev",
+        "silverstripe/recipe-content-blocks": "2.10.x-dev",
+        "silverstripe/recipe-solr-search": "2.10.x-dev",
 
-        "silverstripe/registry": "2.x-dev",
-        "silverstripe/totp-authenticator": "4.x-dev",
-        "silverstripe/mfa": "4.x-dev",
-        "silverstripe/crontask": "2.x-dev",
-        "silverstripe/gridfieldqueuedexport": "2.x-dev",
-        "silverstripe/ldap": "1.x-dev",
+        "silverstripe/registry": "2.4.x-dev",
+        "silverstripe/totp-authenticator": "4.2.x-dev",
+        "silverstripe/mfa": "4.5.x-dev",
+        "silverstripe/crontask": "2.4.x-dev",
+        "silverstripe/gridfieldqueuedexport": "2.6.x-dev",
+        "silverstripe/ldap": "1.3.x-dev",
 
-        "silverstripe/textextraction": "3.x-dev",
-        "silverstripe/realme": "4.x-dev",
-        "silverstripe/ckan-registry": "1.x-dev",
-        "silverstripe/webauthn-authenticator": "4.x-dev",
-        "silverstripe/security-extensions": "4.x-dev",
-        "silverstripe/subsites": "2.x-dev",
-        "silverstripe/html5": "2.x-dev",
+        "silverstripe/textextraction": "3.3.x-dev",
+        "silverstripe/realme": "4.2.x-dev",
+        "silverstripe/ckan-registry": "1.4.x-dev",
+        "silverstripe/webauthn-authenticator": "4.4.x-dev",
+        "silverstripe/security-extensions": "4.2.x-dev",
+        "silverstripe/subsites": "2.5.x-dev",
+        "silverstripe/html5": "2.2.x-dev",
 
-        "cwp/starter-theme": "3.x-dev",
-        "cwp/agency-extensions": "2.x-dev",
-        "cwp/watea-theme": "3.x-dev",
-        "silverstripe/recipe-ccl": "2.x-dev",
-        "cwp/cwp": "2.x-dev",
-        "cwp/cwp-pdfexport": "1.x-dev",
-        "dnadesign/silverstripe-elemental-userforms": "3.x-dev",
-        "symbiote/silverstripe-multivaluefield": "5.x-dev",
-        "tractorcow/silverstripe-fluent": "4.x-dev"
+        "cwp/starter-theme": "3.2.x-dev",
+        "cwp/agency-extensions": "2.6.x-dev",
+        "cwp/watea-theme": "3.1.x-dev",
+        "silverstripe/recipe-ccl": "2.9.x-dev",
```

Comment:
```suggestion
        "silverstripe/recipe-ccl": "2.10.x-dev",
```

Reply:
Can we move this requirement with tho other recipes?

Reply:
Updated